Gemfile

```diff
@@ -31,8 +31,8 @@ gem 'will_paginate_mongoid', "~>2.0"
 gem 'rdiscount'
 gem 'nokogiri', "~>1.6.8"
 
-gem 'tire', "0.6.2"
-gem 'tire-contrib'
+gem 'elasticsearch', '~> 0.4'
+gem 'elasticsearch-model', '~> 0.1.8'
```

Comment:
We could make the upgrade; but, I don't think it's worthwhile given the desire to jump to ES 1.x.